app/services/suspend_account_service.rb

```diff
@@ -17,8 +17,8 @@ def purge_user
   end
 
   def purge_content
-    @account.statuses.reorder(nil).find_in_batches do |statuses|
-      BatchedRemoveStatusService.new.call(statuses)
+    @account.statuses.reorder(nil).find_in_batches(batch_size: 100).with_index(1) do |statuses, batch_index|
```

Comment:
probably should be with_index(0), so the first push has no delay? otherwise it will take (# of followers) seconds before they even see the first statuses deleted. 

If it takes someone with 2000 followers 30 minutes before they see the *first* status deleted after trying to delete their account, that's probably a bad idea.